src/components/Navigation.js

```diff
@@ -1,35 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './navigation.css';
 import { NavLink } from 'react-router-dom';
+import Hamburger from 'hamburger-react';
 import logo from '../images/icons/logo.png';
```

Comment:
In terms of applying React best practices, It's better to import react core files at first, then logic, then components, and at the end styles and images.